packages/staking/src/features/store/delegationPortfolioStore/useDelegationPortfolioStore.ts

```diff
@@ -1,16 +1,10 @@
 import { Wallet } from '@lace/cardano';
 import { create } from 'zustand';
 import { immer } from 'zustand/middleware/immer';
-import { CARDANO_COIN_SYMBOL, LAST_STABLE_EPOCH, targetWeight } from './constants';
+import type { ExecuteCommand, Handler } from './stateMachine/types';
```

Comment:
```suggestion
import type { ExecuteCommand, Handler } from './stateMachine';
```

However, I am curious was there somethis wrong with importing those types without the `type` import?

Reply:
in this specific case there probably isn't much of a benefit, in general it helps the compiler as it is obvious such imports can be dropped, also runtime cyclic dependencies can be avoided this way, especially if some file with type definitions, but used solely for types happens to have a non-type import for whatever reason.

But I agree in this specific case it makes not that much sense, it was rather my "reflex" to isolate type imports, will merge it into a single one

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md